react/components/Dots.js

```diff
@@ -5,7 +5,7 @@ import debounce from 'debounce'
 import EventListener from 'react-event-listener'
 import { constants } from '../utils'
 import resolveSlidesNumber from '../utils/resolveSlidesNumber'
-import styles from './styles'
+import styles from './styles.css'
```

Comment:
The only change on this file was this (add the `.css` @ the end of import)... The rest was just `prettier`, avoid reviewing those.